node-data/src/ledger.rs

```diff
@@ -323,20 +326,35 @@ impl PartialEq<Self> for SpentTransaction {
 
 impl Eq for SpentTransaction {}
 
```

Comment:
Could use a comment:

```suggestion
/// Includes a `NilQuorum` certificate and the key of the expected block generator
```